packages/devtools_app/lib/src/debugger/debugger_state.dart

```diff
@@ -118,27 +118,27 @@ class DebuggerState {
     });
   }
 
-  Future<void> addBreakpoint(String scriptId, int line) {
+  Future<dynamic> addBreakpoint(String scriptId, int line) {
     return _service.addBreakpoint(isolateRef.id, scriptId, line);
   }
 
-  Future<void> addBreakpointByPathFragment(String path, int line) async {
+  Future<dynamic> addBreakpointByPathFragment(String path, int line) async {
     final ScriptRef ref =
         scripts.firstWhere((ref) => ref.uri.endsWith(path), orElse: () => null);
     if (ref != null) {
       return _service.addBreakpoint(isolateRef.id, ref.id, line);
     }
   }
 
-  Future<void> removeBreakpoint(Breakpoint breakpoint) {
+  Future<dynamic> removeBreakpoint(Breakpoint breakpoint) {
     return _service.removeBreakpoint(isolateRef.id, breakpoint.id);
   }
 
-  Future<void> setExceptionPauseMode(String mode) {
+  Future<dynamic> setExceptionPauseMode(String mode) {
     return _service.setExceptionPauseMode(isolateRef.id, mode);
   }
 
-  Future<Stack> getStack() {
+  Future<dynamic> getStack() {
```

Comment:
this should continue to return Future<Stack> 
We should return null or an empty Stack object if a Sentinel is returned. Code using this API is not expecting Sentinel objects and we are hiding the bug by returning dynamic. Alternately we could return Object here and elsewhere and then fix all the upstream analysis errors. Likely it is simpler to fix them all up within the State objects. users of the State object shouldn't need to know about the VM Sentinel concept.

Reply:
done